lib/src/js/polymer_base_dart.html

```diff
@@ -19,6 +19,30 @@
       if (this.__dartDetached__) this.__dartDetached__();
     };
 
+    var originalDeserialize = Polymer.Base.deserialize;
```

Comment:
You want me to just rip some of that out in the process or leave that for you to cleanup?